packages/sfui/frameworks/react/shared/utils/props.ts

```diff
@@ -16,7 +15,7 @@ while user can pass anything. Sometimes your props getter might have to make ext
 interface UserProps<T> extends AllHTMLAttributes<T>, RefAttributes<T> {}
 
 export function createPropsGetter<TProps>(resolver: (userProps: UserProps<HTMLElement>) => TProps) {
-  return function resolve<TUserProps = {}>(userProps: TUserProps = {} as TUserProps): Prettify<TProps & TUserProps> {
+  return function resolve<TUserProps = {}>(userProps: TUserProps = {} as TUserProps): TProps & TUserProps {
```

Comment:
Wasn't this `Prettify` useful here?

Reply:
Its useful - yes, but causes circular dependency issue and its not possible to build nextjs library on local